pkg/terraform/block.go

```diff
@@ -299,7 +299,7 @@ func (b *Block) GetAttribute(name string) *Attribute {
 	return nil
 }
 
-func (b *Block) GetNestedAttribute(name string) *Attribute {
+func (b *Block) GetNestedAttribute(name string) (*Attribute, *Block) {
```

Comment:
Now this function returns the parent attribute block for more accurate metadata output.